source/frontends/libretro/environment.h

```diff
@@ -20,5 +20,6 @@ namespace ra2
 #define MAX_PADS 1
 
   void display_message(const std::string & message);
+  void display_message_alt(const std::string & message);
```

Comment:
what is the reason for this function only the different duration?

Reply:
An oversight of mine. Now integrated into the display_message().